src/app/jumio/jumio.component.ts

```diff
@@ -47,7 +47,10 @@ export class JumioComponent implements OnInit, OnDestroy {
   }
 
   getJumioURL(routeSuffix: string): string {
-    const url = new URL(`${window.location.origin}/${routeSuffix}`)
+    // Jumio URLs cannot be localhost, so default to bitclout.com if localhost.
+    const origin = window.location.origin.startsWith("http://localhost") ? "https://bitclout.com" : window.location.origin;
```

Comment:
Idk what the ramifications are of this, but I could trick this code into setting bitclout.com as the origin by registering a domain like localhostfun.com. Is there a way to make this more robust so that this isn't possible, just to eliminate any edge-cases that could cause?

Reply:
Sure we can make it more robust but all that happens if you trick this function is that you get redirected to bitclout.com instead of the local host url so I don't think it's a huge deal. This case is only on there to support running a local identity